src/publisher/pubsub-message.ts

```diff
@@ -55,6 +55,23 @@ export interface PubsubMessage
    * @private
    */
   publishSchedulerSpan?: tracing.Span;
+
+  /**
+   * If this is a message being received from a subscription, expose the ackId.
```

Comment:
I think expose here just means expose to the other parts of the client library and not users right?

Reply:
Yep. I need to make a whole separate pass of converting @private to @internal since tsdoc changed that.